modules/custom/openy_activity_finder/templates/openy-activity-finder-program-search-page.html.twig

```diff
@@ -133,69 +135,118 @@
+                      </div>
+                      {# if we know that registration is NOT available #}
+                      <div v-if="item.availability_status == 'closed'">
+                        <span class="availability-note">${ item.availability_note }</span>
+                      </div>
+                      </div>
+                    <div v-if="item.price.length > 0" v-html="item.price" class="item-price"></div>
+                    <div v-else class="col-12 col-xs-12">
+                      <span class="availability-note"><i>{{ 'Click on \'Check Availability\' for pricing'|t }}</i></span>
+                    </div>
                   </div>
                 </div>
               </div>
 
-              <div v-show="table.length == 0 && !loading" class="schedules-empty_results">
-                <i class="fa fa-exclamation-circle"></i>
-                <span>{{ 'Sorry, no results were found with your search criteria. Please try another selection.'|t }}</span>
+              <div v-show="table.length == 0 && !loading" class="af-schedules-empty_results">
+                <h2>{{ 'Oh no!'|t }}</h2>
+                <br/>
+                <p>{{ 'We\'re sorry, but no results meet your search criteria'|t }}</p>
+                <br/>
+                <br/>
+                <p><strong>{{ 'What criteria is most important to you?'|t }}</strong></p>
+                <a class="btn" href="#" v-on:click.stop.prevent="searchAlternativeResults('day')">{{ 'Day'|t }}</a><br/>
+                <a class="btn" href="#" v-on:click.stop.prevent="searchAlternativeResults('program')">{{ 'Program'|t }}</a><br/>
+                <a class="btn" href="#" v-on:click.stop.prevent="searchAlternativeResults('location')">{{ 'Location'|t }}</a><br/>
               </div>
 
             </div>
           </div>
+          <div v-show="table.length > 0 && !loading" class="pager-wrapper">
+            <div v-if="typeof pager_info == 'undefined'">
+              <a v-if="(typeof pages[current_page - 1] != 'undefined' && pages[current_page - 1] != '') || current_page > 1" href="#" v-on:click.stop.prevent="loadPrevPage()" class="btn btn-primary">{{ 'Prev'|t }}</a>
+              <a v-if="typeof pages[current_page + 1] != 'undefined' && pages[current_page + 1] != ''" href="#" v-on:click.stop.prevent="loadNextPage()" class="btn btn-primary">{{ 'Next'|t }}</a>
+            </div>
+            <div v-else>
+
+
+              <div  v-if="pager_info.total_pages > 1" class="pager-controls">
+                <a v-if="current_page > 0" href="#" v-on:click.stop.prevent="loadPageNumber(1)" class="btn btn-primary">{{ '|<'|t }}</a>
+                <a v-if="current_page > 0" href="#" v-on:click.stop.prevent="loadPrevPage()" class="btn btn-primary">{{ '<'|t }}</a>
+
+                <div class="page-of">Page ${ current_page } of ${ pager_info.total_pages }</div>
+
+                {#<ul v-if="pager_info.pages">
```

Comment:
can we remove commented code for not confusing us in the future?

Reply:
done